comps/comps-pulpcore-el8.xml

```diff
@@ -139,6 +139,7 @@
       <packagereq type="default">python38-pulp-ansible</packagereq>
       <packagereq type="default">python38-pulp-certguard</packagereq>
       <packagereq type="default">python38-pulp-cli</packagereq>
+      <packagereq type="default">python38-pulp-cli-deb</packagereq>
```

Comment:
Not 100% sure what this does or if it is needed.

Reply:
a similar change is needed to `comps-pulpcore-el7.xml`

Reply:
Done.